addons/stock/models/stock_quant.py

```diff
@@ -527,6 +527,25 @@ def action_set_inventory_quantity_to_zero(self):
         self.inventory_diff_quantity = 0
         self.inventory_quantity_set = False
 
+    def action_set_inventory_quantity_set_zero(self):
```

Comment:
action_set_inventory_quantity_to_zero

Reply:
the same method is available so we can rename it with `action_set_inventory_quantity_zero`